lib/kernel/src/trace.erl

```diff
@@ -915,10 +912,6 @@ Set trace pattern for _message sending_.
 Must be combined with `process/4` or `port/4` to set the `send` trace flag for
 one or more processes or ports.
 
-By default all messages sent from `send` traced processes are traced. Use
-`trace:send/3` to limit traced send events based on the message content, the
-sender and/or the receiver.
-
```

Comment:
Why did you remove this?

The thing I'm trying to explain is that, in contrast to call tracing, send and receive tracing are **enabled by default**. It's enough to call `trace:process(Pid, true, [send,'receive'])`. You don't need to call `trace:send` or `trace:receive` unless you want to filter tracing with a match spec based on message content, sender, and/or receiver. If not stated this can be perceived as counter intuitive, I think. 

The reason for this is backward compatibility. The `send` and `'receive'` options to `erlang:trace/3` were introduced long before it was possible to call `erlang:trace_pattern(send|receive, MatchSpec, _)`.

Reply:
I misunderstood. I will add it back as soon as RC3 has been released.